l5kit/Pipfile

```diff
@@ -8,12 +8,12 @@ pytest = "*"
 mypy = "*"
 pytest-cov = "*"
 flake8 = "*"
-isort = "*"
+isort = "==5.6.2"
```

Comment:
why pinning this?

Reply:
like black, these things change and might silently break CI.

Reply:
yes please, we have tests in place just to catch those issues (the ones without the lock) :)